examples/build_file_generation/WORKSPACE

```diff
@@ -67,6 +67,11 @@ local_repository(
     path = "../..",
 )
 
+local_repository(
```

Comment:
Can we add a comment on what local_repository does and how you don't want to use it when you create your own project?